### PR TITLE
Created a Categories CollectionViewCell

### PR DESCRIPTION
**Pull request by doctor3w**
Natasha asked me to create a cell with a UILabel in it to show categories. I didn't have the design so I only got one look at what she wanted when she showed me but it was pretty simple and I think this is what she wanted. 

Change the cell category title by changing categoryName.

### CODE REVIEW

File: Podcast/CategoriesCollectionViewCell.swift [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    //
+    // Mark: Init
+    //
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        backgroundColor = UIColor.red
+        
+        let yPos = (frame.size.height-labelHeight)/2
+        let width = frame.size.width-2*labelPadding
+        categoryNameLabel = UILabel(frame: CGRect(x: 4,y: yPos,width: width,height: labelHeight))
```
Comment (anniexcheng) [inline Podcast/CategoriesCollectionViewCell.swift]: nitpicking, but would love spaces after commas for readability (x: 4, y: yPos, width: width, height: labelHeight)

Comment (anniexcheng) [inline Podcast/CategoriesCollectionViewCell.swift]: Use labelPadding instead of 4

**Comment by anniexcheng** [comment]
@natashaarmbrust is this what you wanted?

**Comment by doctor3w** [comment]
Made your changes @acheng96 
Also Natasha asked me to add a gitignore for the repo, should I add that in this PR or should I make a separate one that will be merged to master?

**Comment by anniexcheng** [comment]
Let's make that a separate one but once @natashaarmbrust gives you the go for this PR, feel free to merge it in!

**Comment by doctor3w** [comment]
Okay sounds good to me! Thanks!

**Comment by anniexcheng** [comment]
@natashaarmbrust can Drew merge this in?

**Comment by natashaarmbrust** [comment]
I put his file in my code .. this was just a exercise to get him started and looking at our code. He is doing the profile now which is totally separate. Meaning I think we should just delete this branch and PR since itll be merged in with mine

**Comment by anniexcheng** [comment]
Great, I'll just delete this branch and PR then.

**Pull request closed without merging**